File: api/python/examples/pub.py
#encoding=utf8
from zbus import SingleBroker, Producer, Message

broker = SingleBroker(host='localhost', port=15555)

#Producer是轻量级对象,不需要关闭
p = Producer(broker=broker, mq='MyPubSub') 
 
msg = Message()  
msg.set_topic('sse')  
msg.set_body('hello world, from python') 
print p.send(msg)

broker.destroy()

File: api/nodejs/examples/pub.js
var zbus = require("../zbus");
var Message = zbus.Message;
var MessageClient = zbus.MessageClient;
var Producer = zbus.Producer; 

var client = new MessageClient("127.0.0.1:15555");

client.connect(function(){
    var pub = new Producer(client, "MyPubSub");
    var msg = new Message();
	msg.setTopic('sse');
    msg.setBody("hello world from node.js");
    
    pub.send(msg, function(res){
        console.log(res.toString()); 
    });
});


